agenta-backend/agenta_backend/services/docker_utils.py

```diff
@@ -246,7 +246,7 @@ def delete_image(docker_id: str):
         RuntimeError: _description_
     """
     try:
-        client.images.remove(docker_id)
+        client.images.remove(image=docker_id, force=True)
```

Comment:
why do we need the force here?

Reply:
we can remove that, I was actually experimenting if force would get the image deleted even while having running containers.